src/modules/NtpClient.rb

```diff
@@ -282,6 +296,38 @@ def GetNtpServersByCountry(country, terse_output)
       end
     end
 
+    def read_ntp_conf
+      if !File.exist?(NTP_FILE)
```

Comment:
one question. If it is used in installation, it will read file from insts-sys and not target system, which we probably do not want. In such case I would like to point you to two yast classes - https://github.com/yast/yast-yast2/blob/5762181d62762816a73fc040362c1efb5d97deed/library/general/src/modules/FileUtils.rb#L54 and https://github.com/yast/yast-yast2/blob/5762181d62762816a73fc040362c1efb5d97deed/library/system/src/lib/yast2/target_file.rb#L54